frontend/src/containers/LoginPage/styles.css

```diff
@@ -0,0 +1,3 @@
+.root {
```

Comment:
Is using `.root` a good choice here? Maybe it would be better to make this style specific to a className `side-image`?

Reply:
This style is for the root grid element. That's why it's named root. It's not for the side-image